src/components/helpcenter/HelpCenterMenuContent.vue

```
@@ -68,26 +84,30 @@
+        class="section-description flex items-center gap-2.5 self-stretch px-8 pt-2 pb-2"
+      >
+        {{ $t('helpCenter.whatsNew') }}
+      </h3>
 
       <!-- Release Items -->
-      <div
-        v-if="hasReleases"
-        role="group"
-        :aria-label="$t('helpCenter.recentReleases')"
-      >
+      <div v-if="hasReleases" role="group" aria-label="Recent releases">
```

ESLint

[error] raw text 'Recent releases' is used

(@intlify/vue-i18n/no-raw-text)